radio_test/rx/main_radio_test_rx.c

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @defgroup project_Radio    Radio RX test application
+ * @ingroup projects
+ * @brief This is the radio rx code for radio test
+
+ *
+ * @author Raphael Simoes <raphael.simoes@inria.fr>
+ * @copyright Inria, 2024
+ */
+
+#include <nrf.h>
+#include <stdlib.h>
+#include <string.h>
+// Include BSP headers
+#include "uart.h"
+#include "gpio.h"
+#include "board_config.h"
+#include "radio.h"
+#include "timer.h"
+
+uint8_t write_crc[3] = "CRC";
+uint8_t uart_end     = '\x00';
```

Comment:
(already commented about in my previous review)

```suggestion
static const uint8_t write_crc[3] = "CRC";
static const uint8_t uart_end     = 0x00;
```